packages/realm/rollup.config.mjs

```diff
@@ -92,6 +93,38 @@ export default [
     ],
     external: ["bson", "debug", "react-native"],
   },
+  {
+    input: "src/browser/index.ts",
+    output: {
+      file: mainExport.browser,
+      format: "es",
+      sourcemap: true,
+    },
+    plugins: [
+      nodeResolve({
+        mainFields: ["browser", "module", "main"],
+        resolveOnly: ["@realm/network-transport", "path-browserify"],
+      }),
+      // We need to use `commonjs` because of "path-browserify"
+      commonjs(),
+      replace({
+        preventAssignment: true,
+        delimiters: ["", ""],
+        values: {
+          '"../generated/ts/native.mjs"': '"../generated/ts/native-browser.mjs"',
+        },
+      }),
+      typescript({
+        tsconfig: "src/browser/tsconfig.json",
+        noEmitOnError: true,
+        outputToFilesystem: true,
+      }),
+      copy({
+        targets: [{ src: "./generated/ts/realm-js-wasm.wasm", dest: "./dist/" }],
+      }),
```

Comment:
Can we instead just generate to the right place? I think this file is now being copied twice after generation which seems silly.

Reply:
With the change introduced in https://github.com/realm/realm-js/pull/5497#discussion_r1121464986 the `realm-js-wasm.was & realm-js-wasm.js` will be generated from the `bindgen` without an extra copy step in cmake. These files will be under `packages/realm/generated` which is already used by other templates (`node`/`react-native`). 
Rollup is already bundling `realm-js-wasm.js` but it's not aware of the wasm file, hence this manual copy to `dist`